src/CosmosClient.ts

```diff
@@ -25,14 +63,38 @@ export class CosmosClient {
     );
   }
 
-  public async getDatabaseAccount(): Promise<Response<DatabaseAccount>> {
-    return this.documentClient.getDatabaseAccount();
+  /**
+   * Get information about the current {@link DatabaseAccount} (including which regions are supported, etc.)
+   */
+  public async getDatabaseAccount(): Promise<CosmosResponse<DatabaseAccount, CosmosClient>> {
+    const response = await this.documentClient.getDatabaseAccount();
+    return { body: response.result, headers: response.headers, ref: this };
   }
 
+  /**
+   * Used for reading, updating, or deleting a existing database by id or accessing containers belonging to that database.
+   *
+   * This does not make a network call. Use `.read` to get info about the database after getting the {@link Database} object.
+   *
+   * @param id The id of the database.
+   * @example Create a new container off of an existing database
+   * ```typescript
+   * const container = client.database("<database id>").container("<container id>");
```

Comment:
This example text says "create" so I would change this to:

`client.database("<database id>").containers.create("<container id>")`

Or change the example text

Reply:
Ah, good catch.